index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page Dashboard</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="main">
+        <div class="header-main">
+            <div class="header-container">
+                <div class="header">
+                    <div class="header-logo">Header Logo</div>
+                    <div class="header-links">
+                        <a>Header link 1</a>
+                        <a>Header link 2</a>
+                        <a>Header link 3</a>
+                    </div>
+                </div>
+                <div class="header-main-section">
+                    <div class="header-description"> 
+                        <div class="hero-main-text">This website is awesome</div>
+                        <div class="hero-secondary-text">
+                            This website has more subtext that goes under the main title. It's a smaller font and the color is lower contrast
+                        </div>
+                        <button class="sign-up-btn">Sign Up</button>
+                    </div>
+                    <img src="./images/header-image.jpg" alt="Shot of buildings"/>
+                </div>
+            </div>
+            <div class="info-container">
+                <div class="info-heading">
+                    Some Random Information
+                </div>
+                <!-- -->
+                <div class="info-images">
+                    <figure class="item">
+                        <img src="./images/info-1.jpg" alt="Image with a quote"/>
+                        <figcaption class="caption">This is some subtext under an illustration or an image</figcaption>
+                    </figure>
+                    <figure class="item">
+                        <img src="./images/info-2.jpg" alt="image with another quote"/>
+                        <figcaption class="caption">This is some subtext under an illustration or an image</figcaption>
+                    </figure>
+                    <figure class="item">
+                        <img src="./images/info-3.jpg" alt="Image with books on a shelves"/>
+                        <figcaption class="caption">This is some subtext under an illustration or an image</figcaption>
+                    </figure>
+                    <figure class="item">
+                        <img src="./images/info-4.jpg" alt="Image of code in HTML"/>
```

Comment:
Can you include the `height` and `width` of the image
because --
_When these values aren’t included, if an image takes longer to load than the rest of the page contents, the image won’t take up any space on the page at first, but will suddenly cause a drastic shift of the other page contents once it does load in. Explicitly stating a height and width prevents this from happening, as space will be “reserved” on the page and will just appear as a blank space until the image loads._